app/views/posts/_message.html.haml

```diff
@@ -0,0 +1,3 @@
+.row.alert.alert-success
```

Comment:
I'm not sure about the use of .alert.alert-success here -- it's not really semantically right.  Semantically, alert-info might be more appropriate, or we might want to create a class of our own for this sort of contextualising information, since we're starting to use it in a few different places.

I'm also concerned that (based on the screenshot) the links within this message aren't visibly different from the surrounding text.

Finally, I think "message" is a bit of an over-general name for this partial, and since it's only a couple of lines, I'd be fine with it being included in the main show.html.haml rather than being broken out into a partial at all.